src/views/bar.js

```diff
@@ -0,0 +1,59 @@
+//bar chart
+
+let barChart = document.getElementById("myBarChart")
+let xVal = ["swift", "C/C++", "C#", "Golang", "Rust", "Python", "javascript", "typescript", "PHP", "Bash", "Go", "Groovy", "VBA", "dart", "Ada", "Scalar", "Java"];
```

Comment:
Now this data should be coming from the [survey.yaml](https://github.com/osscameroon/devsurvey/blob/main/src/res/survey.yaml) file. We should not write them ourselves.